tests/test_Functional_API.py

```diff
@@ -18,4 +18,9 @@ def test_get_api_version(self):
         zapi = ZabbixAPI(url='http://127.0.0.1',
                          user='Admin',
                          password='zabbix')
-        self.assertEqual(zapi.api_version(), '3.0.1')
+        if os.environ['ZABBIX_VERSION'] == '3':
+            self.assertEqual(zapi.api_version(), '3.0.1')
+        elif os.environ['ZABBIX_VERSION'] == '2':
+            self.assertEqual(zapi.api_version(), '2.4.7')
```

Comment:
Could we run test also on version 2.2.4 or more older? There was a changes in api between this two versions.